web/src/input_pill.ts

```diff
@@ -344,8 +344,10 @@ export function create<T>(opts: InputPillCreateOptions<T>): InputPillContainer<T
                 e.key === "Backspace" &&
                 (funcs.value(e.target).length === 0 || window.getSelection()?.anchorOffset === 0)
             ) {
-                e.preventDefault();
-                funcs.removeLastPill();
+                if (funcs.value(e.target).length === 0) {
```

Comment:
Can't we directly update the above if-condition to remove `window.getSelection...` condition?

Reply:
Yes @sahil839 we can. I added an if condition just to preserve the original structure. Should I update it?

Reply:
Yes, of course you should prioritize how the code reads after your changes over preserving some existing structure chosen with different code in mind.